### PR TITLE
Remove whitespace in DN

### PR DESCRIPTION
**Pull request by sebas-intellegens**
### Description
According to https://opensearch.org/docs/latest/security-plugin/configuration/security-admin/#configure-the-admin-certificate
there shouldn't be any whitespace in the distinguished name, so I assume it was not correct here?

### Check List
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by DandyDeveloper — APPROVED** [review]

**Comment by TheAlgo** [comment]
1 small thing @sebas-intellegens you missed updating the Changelog

File: charts/opensearch/CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Deprecated
 ### Removed
 ### Fixed
+- Remove whitespace from `admin_dn` in base `values.yaml`.
```
Comment (peterzhuamazon) [inline charts/opensearch/CHANGELOG.md:14]: Wrong edit, this should be in a new block and I recommand you use 1.3.2 since #128 is using 1.3.1.

Thanks.

**Review by peterzhuamazon — CHANGES_REQUESTED** [review]

File: charts/opensearch/CHANGELOG.md (hunk 1 above)
Comment (peterzhuamazon) [reply charts/opensearch/CHANGELOG.md:14]: Probably need to change to 1.4.1 later as #123 is having a bump to 1.4.0 for breaking changes.

Comment (sebas-intellegens) [reply charts/opensearch/CHANGELOG.md:22]: https://github.com/opensearch-project/helm-charts/pull/130/commits/c1663155b754fdb3e82e507b1c2874db88dbfdf4

**Comment by peterzhuamazon** [comment]
Hi @sebas-intellegens you need to pull from origin/main as we have merged several PRs already.
Conflicts with files.

Thanks.

**Review by peterzhuamazon — APPROVED** [review]
Thanks.

**Pull request merged**